src/server/pps/server/api_server.go

```diff
@@ -760,6 +767,16 @@ func (a *apiServer) listJob(
 	jobs := a.jobs.ReadOnly(ctx)
 	jobInfo := &pps.JobInfo{}
 	_f := func(string) error {
+		if number == 0 {
+			return errutil.ErrBreak
+		}
+		if paginationMarker != nil {
+			createdAt := time.Unix(int64(jobInfo.Created.GetSeconds()), int64(jobInfo.Created.GetNanos())).UTC()
+			fromTime := time.Unix(int64(paginationMarker.GetSeconds()), int64(paginationMarker.GetNanos())).UTC()
+			if createdAt.Equal(fromTime) || !reverse && createdAt.After(fromTime) || reverse && createdAt.Before(fromTime) {
```

Comment:
so we don't want to include records that were created at `fromTime`?

Reply:
nvm that makes sense based on the usage

Reply:
yeah, the usage pattern is that users will get the first page of results and then request the next page by specifying the paginationTimestamp as the createdAt time of the last job in the first page. We don't want to return that last job again so we exclude matching timestamps